dawn/package.json

```diff
@@ -73,6 +76,7 @@
     "seedrandom": "2.4.2",
     "smalltalk": "2.2.0",
     "ssh2": "0.5.4",
-    "superagent": "3.4.1"
+    "superagent": "3.4.1",
+    "ws": "^2.2.3"
```

Comment:
These package installs should ideally be exact versions (without any `^` or `=` prefix). I believe `yarn add [pkg] --exact` (or `-E`) will do what you want.